### DISCUSSION TITLE
Self-promotion and self-marketing for RSEs (Session one - Room one)

### DISCUSSION THREAD

**Discussion by jduckles** [RSEAA/RSEAA.github.io#128] [Discussion topics for #RSEAA24]
A discussion and potential session around self-promotion and marketing your work as an RSE. 

When I was at the University of Oklahoma, I built traction in my work as an RSE / Informatics professional by asking those I worked with for specific feedback about the work I did with them and how it helped them in their research. I built what I began to call a "brag book" of all the grants, projects I had worked on and specific written feedback from my collaborators about MY contribution to their research goals. This document, along with similar documents from my colleagues, was used to raise the visibility of our work with the leadership of the University. Ultimately we could translate this into fractional salary funding for an Informatics Group within Libraries and IT and use it to support a continued case for our involvement and particpation in informatics-heavy grant writing. This became a self-fulfilling point-of-difference in our proposals creating a growing stream of new revenue and new collaborations. 

While each case / institution is different, I think a common thing that RSEs might need support with is this kind of self-marketing of themselves and their work.

I would love to craft flexible session with small-group breakouts where participants actually spend the session crafting the outline for their own Brag Book. 

From a brag book you can:

1. Ask for more institutional support
2. Self-market on your personal website 
3. Update your CV with more specific details, emulating language your collaborators use to describe you.
4. And more! 

Does anyone else do this? How has it gone for you?

Would there be interest in a session to help you get over the hump and begin building your own, or updating one you haven't looked at in a long time?

**Comment by rowlandm** [comment]
Great idea Jonah!!!!

**Comment by jduckles** [comment]
What are some ways you've seen yourself others self-promote as developers or RSEs?

**Reply by jduckles** [reply]
*Notes from the breakout session*
- Posting on linkedin about their projects, or blogs. 
- RSEs create software that is very specific to a particular research area. It can be hard to translate that value to other research areas. Software is used by small niche areas. 
- have done lots of stuff in 20 years, but never made a list of what they had done. Was really nice to pull together a bunch of things that they had worked on in the past. 
- Using researcher identifier (ORCID) to link together everything
- A team was built to tackle computational things, was created as a centre of excelence and it was noted that the software team was an important part of that success. 
- Websites from successful engineers have demonstrations of what they've done.
- Self-promoting self on linkedin. Have been sharing course and achievements on LinkedIn. People showcase themselves by their achievements and certificates. Sharing experiences and even interviews or other kinds of feedback on their profiles. 
- Starting with work to help people identify what an RSE is like, and what kind of people can believe that they could be called an RSE. 
- As a hiring manager, when hiring people, checking in on someone's GitHub page and what kinds of code / documentation they write.
- Verbalizing the number of contacts / tickets etc. 
- Figuring out what's normal in your field and maybe not in other fields. 
- When it isn't quite obvious you can find ways to phrase things to appeal to other groups.
